corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -137,8 +137,8 @@ def _check_initial_state(case):
         self.assertEqual(f3, close.form_id)
 
         # todo: should this be the behavior for archiving the create form?
-        form_acessors = FormAccessors(REBUILD_TEST_DOMAIN)
-        f1_doc = form_acessors.get_form(f1)
+        get_form = XFormInstance.objects.get_form
+        f1_doc = get_form(f1)
```

Comment:
Is the reason you haven't included 'domain' in places like this just because they are in tests and it doesn't really matter?

Reply:
Correct.